code/modules/mob/living/simple_animal/hostile/hellhound.dm

```diff
@@ -121,9 +108,48 @@
 	maxHealth = 400
 	health = 400
 	force_threshold = 5 // no punching
+	universal_speak = 1
 	smoke_freq = 200
 	life_regen_cycle_trigger = 5
 	melee_damage_lower = 20
 	melee_damage_upper = 30
 	environment_smash = 2
 	gold_core_spawnable = NO_SPAWN
+
+/mob/living/simple_animal/hostile/hellhound/greater/New()
+	. = ..()
+	// Movement
+	var/obj/effect/proc_holder/spell/targeted/ethereal_jaunt/shift/shiftspell = new
+	AddSpell(shiftspell)
+	var/obj/effect/proc_holder/spell/targeted/area_teleport/teleport/telespell = new
+	telespell.clothes_req = FALSE
+	telespell.invocation_type = "none"
+	AddSpell(telespell)
```

Comment:
Not sure if a new subtype should be made or modified here.

Probably better to do it this way to avoid clogging up with subtype

Reply:
Leaving as-is then unless you request it be changed.